src/pages/contact-us.js

```diff
@@ -38,7 +38,7 @@ const ContactUs = () => (
         </div>
 
         <EmailLink
-          link="/"
+          link="mailto:hello@neontribe.co.uk"
           background="#5600ee"
           border_color="#5600ee"
           mobileSize="0.9rem"
```

Comment:
If this is the only use of the component, do these need to be passed as props?

Reply:
I was planning on replacing all of the 'Styled Link' components with this Email Link component, so I will keep this in and then replace all of the styled links.